tools/jarcat/jarcat.go

```diff
@@ -128,6 +129,10 @@ var opts = struct {
 	NoDirEntries            bool              `short:"n" long:"nodir_entries" description:"Don't add directory entries to zip"`
 	RenameDirs              map[string]string `short:"r" long:"rename_dir" description:"Rename directories within zip file"`
 	StripBytecodeTimestamps bool              `short:"b" long:"strip_bytecode_timestamps" description:"Strips timestamps from any .pyc / .pyo files encountered."`
+
+	Tar    bool   `long:"tar" description:"Write a tarball instead of a zipfile. Note that most other flags are not honoured if this is given."`
+	Gzip   bool   `short:"z" long:"gzip" description:"Apply gzip compression to the tar file. Only has an effect if --tar is passed."`
+	Prefix string `long:"prefix" description:"Flatten all files into a directory with this name within the tarball."`
```

Comment:
what's the flattening use case? wouldn't it be more common/expected to only prefix the usual tree with the given prefix? You can't run into file collisions then, too